src/Tools/dotnet-monitor/CollectionRules/Options/CollectionRuleDefaultsInterfaces/EgressProviders.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.CollectionRuleDefaultsInterfaces
+{
+    internal interface EgressProviders
```

Comment:
Consider something like IEgressProviderProperties or IEgressProviderSettings. Similiar to the other interfaces.